epictrack-api/src/api/services/task.py

```diff
@@ -241,8 +271,14 @@ def bulk_update(cls, data: dict):
         return "Updated successfully"
 
     @classmethod
-    def bulk_delete_tasks(cls, task_ids: List):
+    def bulk_delete_tasks(cls, task_ids: List, work_id: int):

```

Comment:
@dinesh-aot @saravanpa-aot should we add a "Delete" role for operations like this. We only have Create and Edit now

Reply:
Good Question. I think we should create a role called Delete. @saravanpa-aot thoughts?

Reply:
Added this ticket for the Delete role https://app.zenhub.com/workspaces/epictrack-63891ea941d309001fa292cf/issues/gh/bcgov/epic.track/1679, happy to merge this once approved